### PR TITLE
New React Cheat Sheet

### PR DESCRIPTION
**Pull request by vikashvverma**
**What does your Instant Answer do?**

This instant answer is a quick reference of React, a JavaScript library for building user interfaces. 

**What is the data source for your Instant Answer? (Provide a link if possible)**

[ https://facebook.github.io/react/docs/getting-started.html](https://facebook.github.io/react/docs/getting-started.html)

**Why did you choose this data source?**

This is official source for React.

**Are there any other alternative (better) data sources?**

Since this is the official source, I don't think there is any other better source.

**Which communities will this Instant Answer be especially useful for? (gamer, book lovers, etc)**

Programmers

**Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?**

No

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

screen shot 2016-02-16 at 5 34 36 pm

IA Page: https://duck.co/ia/view/react_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by vikashvverma** [comment]
@GuiltyDolphin @moollaza Can you review this?

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=react cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/react_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=react cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/react_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
> @GuiltyDolphin @moollaza Can you review this?

@vikashvverma We'll get around to reviewing as soon as we can -- there's many other PR's coming in so please be patient ;)

File: share/goodie/cheat_sheets/json/reactjs.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
+{
+    "id": "reactjs_cheat_sheet",
+    "name": "React",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/reactjs.json]: Should probably be "Facebook React"?

**Comment by moollaza** [comment]
@vikashvverma Thanks! This LGTM :+1:

File: share/goodie/cheat_sheets/json/reactjs.json (hunk 1 above)
Comment (vikashvverma) [inline share/goodie/cheat_sheets/json/reactjs.json]: @moollaza Should I rename it to `Facebook React`. What do you say?

**Pull request merged**